kirafan/quiz.ts

```diff
@@ -317,7 +318,19 @@ export default (slackClients: SlackInterface): void => {
       const randomKirafanCard = sample(await getKirafanCards());
       const problem = await generateProblem(randomKirafanCard);
       const quiz = new KirafanAteQuiz(slackClients, problem, postOption);
-      await quiz.start();
+      const result = await quiz.start();
+      if (result.state === 'solved') {
+        await increment(result.correctAnswerer, 'kirafan-answer');
+        if (result.hintIndex === 1) {
```

Comment:
hintIndexはhintMessage(not problem)が投稿されるたびに+1されるので、first-hintのときは0です
```suggestion
        if (result.hintIndex === 0) {
```